### PR TITLE
Fix Jormun version

### PR DESCRIPTION
**Pull request by azime**
Jira: [NAV-2145](https://navitia.atlassian.net/browse/NAV-2145)
On prd env:
image

[NAV-2145]: https://navitia.atlassian.net/browse/NAV-2145?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docker/debian11/Dockerfile-jormungandr [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
 COPY ./source/navitiacommon ./navitiacommon
 COPY ./source/jormungandr ./jormungandr
+COPY ./docker/jormungandr_version.j2 .
 COPY ./docker/ca-certificates/*.crt /usr/local/share/ca-certificates/
 
 RUN apt-get update --fix-missing \
     && apt-get install -y curl libpq5 apache2 python3.9-dev python3-pip git libgeos-c1v5 ca-certificates \
     && update-ca-certificates \
+    && pip3 install --no-cache-dir j2cli==0.3.10 \
+    && JORMUNGANDR_REVISION=${jormungandr_version} j2 jormungandr_version.j2 -o ./jormungandr/jormungandr/_version.py \
```
Comment (pbougue) [inline docker/debian11/Dockerfile-jormungandr]: It bothers me to create a new file after `navitia_version.py.cmake` and use a new tool.
Let's discuss it, as I guess there are reasons for that :balance_scale:

Comment (azime) [reply docker/debian11/Dockerfile-jormungandr]: :+1:  Done

File: .github/workflows/dockers_builder_dev.yml [hunk 2]
```diff
@@ -9,7 +9,6 @@ env:
   jessie_version: debian8
   bullseye_version: debian11
   front_jessie_navitia_components: 'tyr-web instances-configurator'
-  front_bullseye_navitia_components: 'jormungandr'
```
Comment (pbougue) [inline .github/workflows/dockers_builder_dev.yml:12]: Why do you remove this one?

File: docker/debian11/Dockerfile-jormungandr (hunk 1 above)
Comment (pbougue) [reply docker/debian11/Dockerfile-jormungandr]: :heart:

File: .github/workflows/dockers_builder_dev.yml (hunk 2 above)
Comment (azime) [reply .github/workflows/dockers_builder_dev.yml:12]: Unused variable

**Review by xlqian — APPROVED** [review]

**Review by stifoon — APPROVED** [review]

File: .github/workflows/dockers_builder_dev.yml (hunk 2 above)
Comment (pbougue) [reply .github/workflows/dockers_builder_dev.yml:12]: Ah, it's built in this workflow but not managed exactly the same way as other jobs.

**Review by pbougue — APPROVED** [review]

**Pull request merged**